src/test/system_tests/test_build_chunks.js

```diff
@@ -12,25 +12,24 @@ const _ = require('lodash');
 const fs = require('fs');
 const AWS = require('aws-sdk');
 const util = require('util');
-const crypto = require('crypto');
+// const crypto = require('crypto');
```

Comment:
remove

Reply:
Should not be removed. it is used in the commented out tests the ew should re-enable (maybe not as part of the current task).
I can check if my rewrite of the test fixed the bug already.

Reply:
Check it, if it's not working, remove it. We have the history in git, we don't need comments as history

Reply:
We don't have the history it is a rewrite of the tests, If you want we can merge then add a new PR to remove

Reply:
We have the old functions, so we have the idea of the logic of the test. If the new code isn't working, it means someone will have to re-write it which means the current code only value is to understand what was under test, which can also be deduced from the previous code version.